src/hevm/hevm-cli/hevm-cli.hs

```diff
@@ -373,7 +373,7 @@ equivalence cmd =
        Just sig' -> do method' <- functionAbi sig'
                        return $ Just (view methodSignature method', snd <$> view methodInputs method')
 
-     void . runSMTWithTimeOut (solver cmd) (smttimeout cmd) . query $
+     void . runSMTWithTimeOut (solver cmd) (fromMaybe 5000 (smttimeout cmd)) . query $
```

Comment:
if you set the default in `runSMTWithTimeOut` we don't have to set it in two places

Reply:
done.